tests/Dockerfile.crossplane

```diff
@@ -0,0 +1,11 @@
+FROM python:3.12-alpine
+
+ARG NGINX_CONF_DIR
+
+RUN pip install crossplane
```

Comment:
We're not using the go version? Is the python version still maintained?

Reply:
This was the easy one to install. Wasn't sure if the Go one required us to build a binary or if it was just a library.

Reply:
I think it's just a library. Seems like they python one hasn't been updated in the past 2 years.

Reply:
If the python one gets to the point where it doesn't work for us, then we can create a Go binary that uses the library, but probably not worth the effort right now since the python one works.

Reply:
We can't just use crossplane-go as a library? Without the need for another container?

Reply:
We have to run crossplane on the nginx filesystem, so we need a container running in the pod that has crossplane. Neither of our product containers have it (nor should we add it).

Reply:
Updated to use Go crossplane instead of python.